docs/pipeline.rst

```diff
@@ -143,3 +143,41 @@ DataNode
 --------
 .. autoclass:: nvidia.dali.pipeline.DataNode
    :members:
+
+Pipeline Debug Mode (experimental)
+------------------
+
+Pipeline can be run in debug mode by replacing :meth:`@nvidia.dali.pipeline_def <nvidia.dali.pipeline_def>` decorator with its experimental variant
+``@nvidia.dali.experimental.pipeline_def`` and setting parameter ``debug`` to True. It allows you
+to access and modify data inside of the pipeline execution graph as well as use non-DALI data types
+as inputs for DALI operators.
+
+In this mode outputs of operators are of type ``DataNodeDebug`` which is an equivalent to
+:class:`~nvidia.dali.pipeline.DataNode` in the standard mode. You can perform the same
```

Comment:
Here's the same situation as above (`DataNodeDebug` vs `class:~nvidia.dali.pipeline.DataNode`)

Reply:
done